### PR TITLE
LPS 90833 [New Form Builder] The plus icon (repeatable) is overlapping a long label

### PR DESCRIPTION
**Pull request by mateusgomes01**
Solves [LPS-90833](https://issues.liferay.com/browse/LPS-90833) plus some additional refactoring.

Also applies Code Review requests by @natocesarrego

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js [hunk 1]
```diff
@@ -171,6 +171,48 @@ function FieldBase({
 	</>
 	);
 
+	const buttons = repeatable && (
+        <div className="lfr-ddm-form-field-repeatable-toolbar react-field-base-flex-container">
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js:175]: @mateusgomes01, you've renamed the CSS classes, but this renaming wasn't replicated here. Can you please update and retest?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/css/main.scss [hunk 2]
```diff
@@ -640,6 +637,22 @@ p.ddm-label {
...
+}
+
+.field-base-label {
+	word-break: break-word;
+}
+
+.field-base-legend {
+	word-break: break-word;
+}
+
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/css/main.scss:655]: @mateusgomes01, one important thing that I forgot to mention to you: new CSS classes should be added following alphabetic order. :wink: Can you please make sure that this commit follows this rule.

**Review by natocesarrego — CHANGES_REQUESTED** [review]
This pull is almost there. :slightly_smiling_face: Please see my comments below.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js (hunk 1 above)
Comment (mateusgomes01) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js:175]: Sorry, I forgot! Gonna fix it and PR again. Thanks!

**Pull request closed without merging**